web/rest/client/features/provider/retailAccount/putRetailAccount.feature

```diff
@@ -32,3 +32,76 @@ Feature: Update retail accounts
           "outgoingDdi": 3
       }
       """
+
+  @createSchema
+  Scenario: Update a retail account
+    Given I add Retail Company Authorization header
+     When I add "Content-Type" header equal to "application/json"
+      And I add "Accept" header equal to "application/json"
+      And I send a "PUT" request to "/retail_accounts/1" with body:
+      """
+      {
+        "directConnectivity": "yes",
+        "ip": "10.10.10.10",
+        "port": "1010",
+        "ruriDomain": null
+      }
+      """
+     Then the response status code should be 200
+      And the response should be in JSON
+      And the header "Content-Type" should be equal to "application/json; charset=utf-8"
+      And the JSON should be like:
+      """
+      {
+        "id": 1,
+        "directConnectivity": "yes",
+        "ip": "10.10.10.10",
+        "port": 1010,
+        "ruriDomain": null
+      }
+      """
+
+  @createSchema
+  Scenario: Update a retail account
```

Comment:
You've two `Scenario: Update a retail account` in this file. How are they different from each other? Put it in scenario description